config/config_mgmt.py

```diff
@@ -584,7 +593,125 @@ def testRun_Delete_Add_Ports():
+        return
+
+    deps, ret = cm.deletePorts(delPorts=['Ethernet0', 'Ethernet2'], force=True)
+    if ret == False:
+        print("Port Deletion Test failed")
+        return None
+
+    print("\n***Port Deletion Test Passed***\n")
+    import time
+    time.sleep(5)
+
+    portJson = {
+        "PORT": {
+            "Ethernet0": {
+                    "alias": "Eth1/1",
+                    "admin_status": "up",
+                    "lanes": "65,66,67,68",
+                    "description": "",
+                    "speed": "100000"
+            }
+        }
+    }
+
+    #ret = cm.addPorts(ports=['Ethernet0'], portJson=portJson, loadDefConfig=False)
+    ret = cm.addPorts(ports=['Ethernet0'], portJson=portJson, loadDefConfig=True)
+    if ret == False:
+        print("Port Addition Test failed")
+        return None
+
+    print("\n***Port Addition Test Passed***\n")
+
+    return
+
+def testRun_Delete_Add_Ports_3():
+
+    #TODO: Verify config in Config DB  after writing to config DB.
+    print('Test Run Delete Ports')
+    try:
+        cm = configMgmt('configDB', debug=True)
+    except Exception as e:
+        print(e)
+        return
+
+    deps, ret = cm.deletePorts(delPorts=['Ethernet0'], force=True)
+    if ret == False:
+        print("Port Deletion Test failed")
+        return None
+
+    print("\n***Port Deletion Test Passed***\n")
+    import time
+    time.sleep(5)
+
+    portJson = {
+        "PORT": {
+            "Ethernet0": {
+                "alias": "Eth1/1",
+                "description": "",
+                "index": "0",
+                "lanes": "65",
+                "speed": "25000"
+            },
+            "Ethernet1": {
+                "alias": "Eth1/2",
+                "description": "",
+                "index": "0",
+                "lanes": "66",
+                "speed": "25000"
+            },
+            "Ethernet2": {
+                "alias": "Eth1/3",
+                "description": "",
+                "index": "0",
+                "lanes": "67",
+                "speed": "25000"
+            },
+            "Ethernet3": {
+                "alias": "Eth1/4",
+                "description": "",
+                "index": "0",
+                "lanes": "68",
+                "speed": "25000"
+            }
+        }
+    }
+
+    #ret = cm.addPorts(ports=['Ethernet0', 'Ethernet1', 'Ethernet2', 'Ethernet3'], portJson=portJson, loadDefConfig=False)
+    ret = cm.addPorts(ports=['Ethernet0', 'Ethernet1', 'Ethernet2', 'Ethernet3'], portJson=portJson, loadDefConfig=True)
+    if ret == False:
+        print("Port Addition Test failed")
+        return None
+
+    print("\n***Port Addition Test Passed***\n")
+
+    return
+
+def main():
+    from sys import argv
+    if len(argv) > 1:
+        num = argv[1]
+    else:
+        print("Please provide a number 1-3")
+        return
+
     #testRun_Config_Reload_load()
-    testRun_Delete_Add_Ports()
-    #test_sonic_yang()
+    if num == '1':
```

Comment:
Can you add short description of options "1", "2" and "3" for different breakout modes to be tested?

Reply:
I changed the way, It runs tests now.